### ISSUE TITLE
Feature Request : Organ Research Department.

### ISSUE THREAD

**Issue by FuryMcFlurry** [HippieStation/HippieStationdeprecated2020#8698]
Organ Research Department. 

original thread. https://tgstation13.org/phpBB/viewtopic.php?f=9&t=12735

 Long ago i through up for an idea to add in a new department that would allow for another medical research job. or an addition to the genesist

[Implementation]: # How should the feature work out? What exactly is it supposed to do, in your initial idea? If an item, can it be crafted/found/bought and for what?

The idea is that you can grow unique organs and surgery them inside other peoples body.
The system would have tiers of organs and you could unlock better organs via a random button on a machine and scanning that organ to unlock it. ( maybe some other way to implement this or its gonna be an RNG fest)

Organ Research Pod
A machine that scans organs and makes a research plan for organ scanned. must be hooked up with the organ server computer. clicking on this device will remove inserted organ.

Organ Synthesizer
A wondrous machine that creates organs from Synthflesh

ImageOrgan Computer
A computer containing all the buttons for the department the function.
This computer basically operates all the other machines its linked to.
The menu will show you ,
•	Scan organ
-Scans the organ you have in the Organ Research Pod.
•	Research organ
-Opens a Minigame that allows you to gain research points (a currency used to purchase the next tier of organ research), and adds the organ you completed the research with to your database for you to synthesize.
•	View database
-Opens a menu where all your current organs you have completed are listed and ready for creation.
•	Purchase Research 
-Allows you view your current research points and unlock the next tier. Research points can be used to purchase normal limbs and organs data for your database 
•	Synthesize organ
-Creates the selected organ.
•	Synthesize a random organ
-Allows you to select a tier and a random organ from that list will be made, for you to research.

A medical scanning device that allow someone to be scanned and see all their organs within their body. Scanning will give them minor radiation.

Organs and Tumors

Organs are Tiered from bad to good organs. (I want to avoid hidden advantages so healing organs in every part of your body is big no no)

Tier 0 : Tumors
These are mainly bad ones.
Brain Scatter	Deals small brain damage overtime. causes headaches
Hemorrhage	Causes you to cough up blood. Dealing brute damage. 
Acidic	Deals small burn damage overtime. causes stomach pains and makes your vomit randomly
Drugging	Causes hallucinations. makes your head feel like its spinning
Stumbling	Makes you stumble into a random direction, makes you shake uncontrollably. 
Auto-brewery	Slowly injects Ethanol into your bloodstream.
Sleep-inducing	Slowly injects Sleepy toxin/Morphine into your bloodstream. causes yawning
Vomiting	Causes your randomly to vomit.
Black spots	Random black spots appear on your screen.
Tier 1: Gimmicks
Luminescence	Makes you glow in a random color.
Combustion	Sets you aflame randomly
Sanguine consume	Stomach	Drinking blood adds nutrition to you body. also heals you for a tiny amount. 
Squeaky Being hit makes you play a sound
Colored skin	Changes your skin color to a random color 
Brain Healing	Heals your brain damage slowly.
Tier 2
Electric	Randomly electrify an attacker causes sparks to fly to this limb
Random Chem	A random chem is picked and this organ injects you with it ( could be shitty)
Smouldering	Randomly burns an attacker to this limb
Sleep-reducing	Makes you immune to sleep
Cold	Randomly	Cools an attacker
Armored carapace	Adds 5 armor to this limb
Buffed punches Your punches deals extra damage (4 extra brute) with this limb
Dark-vision	Allows you to see into the darkness with these eyes
Tier 3
Shock Immunity	Yellows gloves in limb form
Fire Retardant You cannot be set on fire. with these limbs
Frost Emitting	Your punches will freeze someone with these limbs
Thermal Vision	See mobs through walls with these eyes
Venomous Tail	You can use your stinger ready your next attack to deal tox damage (gives an icon like the krav maga gloves would)

Sprites I have made for this project.
![organ researcher](https://user-images.githubusercontent.com/8770071/44415771-853c2900-a571-11e8-8995-32d797c552bd.gif)
![pod creation](https://user-images.githubusercontent.com/8770071/44415797-9422db80-a571-11e8-8f27-030e85775978.gif)
![organ research new pc](https://user-images.githubusercontent.com/8770071/44415801-971dcc00-a571-11e8-82fb-53b9914b44c5.gif)
![tissue pod](https://user-images.githubusercontent.com/8770071/44415810-9dac4380-a571-11e8-98d5-c6972860a65a.gif)
![tissue pod bubbles](https://user-images.githubusercontent.com/8770071/44415812-9f760700-a571-11e8-81ca-f33262f346dd.gif)
![screenshot_20](https://user-images.githubusercontent.com/8770071/44415844-adc42300-a571-11e8-9ac9-c256ff622776.png)

Everything you read here can be subject to change and better ideas can be spawned forth from this text.

**Comment by xTrainx** [comment]
holy shit thats cool

**Comment by Carbonhell** [comment]
Not sure how to do the black spots one tbh,
No to random chem. It would either be op or shit, and only in a few cases decent. Just code the decent cases.
I do not like the research organ mechanic, maybe use existing organs to increase some sort of biomass point which depends on organ type, rarity and status. Would fit more and be less science-y 
Overall, +1 but needs a few adjustments.

**Comment by Carbonhell** [comment]
Maybe also add some sort of special event to unlock tier 3 organs, or make them very expensive. They look a bit op. Also it could be expanded to special bodyparts too dunno instead of organs only, call it biological research like the one we had.

**Comment by Eagle-Eyess** [comment]
+1 but I feel that most of the organs (especially the memey ones) aren’t good enough. Just feel that better ones need to exist ofc make them hard to obtain tho.

**Comment by Carbonhell** [comment]
@fluxcapacitor1337

**Comment by ghost** [comment]
+1

**Comment by Shdorsh** [comment]
As someone working in healthcare engineering, including cellfree therapy, cell and tissue engineering and therapy, I'd like to add my 2 cents to this:

Can we add a way to extract cells (mesenchymal stem cells, hematopoietic stem cells, differentiated and stem organ cells), a way to amplify them and differentiate them using a complex cell culture medium system? Maybe also adding the possibility of turning differentiated ones into induced pluripotent cells with help of viro and injection of them being able to heal organs and such.

**Comment by praisenarsie** [comment]
![young](https://user-images.githubusercontent.com/31222036/45113800-3a124080-b143-11e8-82b6-b7ff19def8b8.png)
big words for my small mind.

**Comment by ghost** [comment]
yeah that'd be awesome,  make science be actually science with nicely long nerd words instead of "lol gold slime plus plasma make living being lol"

**Comment by Shdorsh** [comment]
Basically, if you want to repair an organ, you do it via healthy cells. In order to get them, you need to drive them out of someone's body, using chemicals named cytokines that make them move out of their normal place and collect them.

Once you got a little bit of stem cells, you can increase their number by what we call amplification: you put the cells in an environment containing nutriments and signals that tell the cells to divide.

Once you got enough stem cells, you will need to turn them into let's say "fully adult" cells, or differentiated cells found in the tissue you want to transplant. Different types of stem cells give different types of cells in the end:  You can make any blood cells with hematopoietic cells for example, while organ stem cells give the type of adult organ cells found in the organ you took from, ofc.

Once differentiated, they can't go back to being young stem cells (unless through inserting genes using a vector like a virus or liposomes) and lose their ability to renew themselves or the entire tissue that the stem cell can turn into, but now fulfill a certain function.

You can't inject yourself stem cells that are not differentiated enough, because it would cause teratocarcinoma, which are benign tumors, apparently especially adult cells that have been turned back into slightly more stem-like cells(induced pluripotent cells).

Also, in case of hematopoietic stem cells, you're basically creating a new immune system in that person's body, so if the stem cells aren't compatible with the person receiving the graft, he'll have a whole new immune system rejecting him, which we call the Graft versus Host disease. Imagine being allergic to yourself. Sounds pretty unfun to be.

**Comment by ghost** [comment]
>someone proposes medical research job
>GIVE IT TO SCIENCE

This clearly belongs in the little patient rooms in medical bay that no one ever uses

**Comment by Carbonhell** [comment]
Accepted but needs futher details about implementation

**Comment by Jujumatic** [comment]
awesome but will need further balancing, still would be a big thing to implement. i would recommend at least $10 bounty
